### PR TITLE
Generate fonts scss

### PR DESCRIPTION
**Pull request by rajsite**
# Pull Request

## 🤨 Rationale

We have been playing with `vite` more and an issue I ran into trying to use it is that it seems to fail to resolve our nimble-tokens `fonts.css` file correctly. See https://github.com/vitejs/vite/issues/12227 and https://github.com/vitejs/vite/issues/9681

I think the root cause is that the font import chain mixed scss and css files:

```
nimble-components/fonts.scss --> nimble-tokens/fonts.css
```

This PR updates the import chain so it only has scss files:

```
nimble-components/fonts.scss --> nimble-tokens/fonts.scss
```

## 👩‍💻 Implementation

Added a build script that generates a `nimble-tokens/dist/fonts/scss/fonts.scss` from the contents of the `fonts.css` file and updated nimble-components to reference it.

Kept the original fonts.css since it is leveraged in environments that don't have good build tooling, i.e. Blazor and the storybook preview.js file.

Updated the vite landing page to reference the correct import from nimble components.

## 🧪 Testing

Verified fonts load in storybook, angular, and landing page builds locally.

## ✅ Checklist

- [x] I have updated the project documentation to reflect my changes or determined no changes are needed.

### CODE REVIEW

File: change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (jattasNI) [inline change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json:1]: Any additional testing we should do when this is uptaken in SLE? If we look at the SystemLinkShared Chromatic build and fonts still work, does that seem sufficient to you?

**Review by jattasNI — APPROVED** [review]

File: change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json (hunk 1 above)
Comment (rajsite) [reply change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json:1]: Yep, that would work. With the angular app in here building I'm pretty confident, but that would be a good double check 👍

Comment (rajsite) [reply change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json:1]: I'll validate in the renovate bubble-up

Comment (rajsite) [reply change/@ni-nimble-components-b677803e-1ef0-4c9d-bffc-9e3d89107467.json:1]: Validated in [Skyline chromatic build](https://ni.visualstudio.com/DevCentral/_build/results?buildId=4635532&view=logs&j=5cb0c4d9-f36d-5fb3-bf2e-85320ef8bd68&t=8b524ba8-3d0d-5872-d8b6-3acaf409afeb)

**Pull request merged**